src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -204,7 +204,7 @@ By combining all the features supported by Portals, you can build really fancy u
 ## Use cases and the plan
```

Comment:
"By combining all the features" should be "By combining all of the features"

Reply:
"can enable a seamless user experience between a website and a third party embed content"

should be

"can enable a seamless user experience between a website and third party embeded content"

Reply:
"Here is the link to the GitHub repo which you can play around with. Be creative and have fun!!

I'd suggest putting this into an `{% Aside %}` so it stands out a bit more. Maybe with more of a call to action like:

```
{% Aside %}
Interested in this demo? [Fork it on GitHub](...) and build your own version!
{% endAside %}
```

Reply:
Done.